src/components/MapChart.vue

```diff
@@ -439,6 +451,7 @@ export default {
         box-shadow: 0 8px 16px 0 rgba(22, 22, 22, 0.12), 0 8px 16px -16px rgba(22, 22, 22, 0.32);
         text-align: left;
         font-size: 0.75rem;
+        pointer-events: none;
```

Comment:
Thanks for this one! 🙏 https://github.com/etalab/covid19-dashboard-widgets/commit/d7651b1305833960a4fde1306e638fa04d4dd197